music-playlist-app/index.html

```diff
@@ -96,7 +116,88 @@ <h1>Music Playlist Project</h1>
             </button>
           </div>
 
-          <div class="playlist-container"></div>
+          <ul class="playlist-container">
+            <li>
+              <div id="song0" class="play-song-btn-container">
+                <button
+                  type="button"
+                  id="we-are-going-to-make-it"
+                  class="song-btn"
+                >
+                  <span class="sr-only">Play</span> We Are Going to Make it
+                  <span class="sr-only">by</span>
+                  - Quincy Larson
+                </button>
+                <div class="btn-container">
+                  <span class="sr-only">Delete We Are Going to Make it</span>
```

Comment:
```suggestion
                  <span class="sr-only">Delete We Are Going to Make it</span>
```

This `span` needs to be inside the `button` text so that the screen reader will announce "Delete We Are Going to Make it" when the user focuses the button. Same for the other four Delete buttons.